docs/UserGuide.md

```diff
@@ -103,17 +103,17 @@ Examples:
 
 Adds a lesson tied to a student with start and end date time.
 
-Format: `lesson add  sid/STUDENTID d/STARTDATE li/LOCATIONINDEX h/DURATION`
+Format: `lesson add sid/STUDENTID f/PRICEPERHOUR d/STARTDATETIME h/DURATION`
 
 * Add a lesson with the student at specified `STUDENTID`.
 * The student ID refers to the index number shown in the displayed person list.
-* The lesson starts at the specified `STARTDATE` in the format `dd-MM-yyyy HH:mm`.
-* The lesson is held at the location specified by the `LOCATIONINDEX`.
-* The location index refers to the index number shown in the displayed location list.
-* The index refers to the index number shown in the displayed lesson schedule.
+* The lesson starts at the specified `STARTDATETIME` in the format `dd-MM-yyyy HH:mm`.
+* The lesson is held at the address of the student.
+* The price per hour refers to the price of the lesson per hour.
+* The price per hour **must be a non-negative integer**.
+* The duration **must be a decimal number or an integer** between 0 and 24 inclusive.
```

Comment:
Although duration is not a field, it is deduced from StartDateTime and EndDateTime.